### DISCUSSION TITLE
导太多了怎么办？

### DISCUSSION THREAD

**Discussion by 0wuming0** [share121/inter-knot#2616] [其他]
求助！最近导太多了，平均一天两次，感觉要死了，有什么破局法吗？

**Comment by shiliuyexiafeng** [comment]
不看，少导，去人多的地方<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/a785e557e39e100ce93d80508f112615_3814701686582051674.png" alt="素裳 冲鸭" width="75" >

**Comment by shxtxdy** [comment]
把你软件卸了网站封了就老实了🌚

**Comment by zhixuan45** [comment]
玩骑马与砍杀2（
转移注意力就行了

**Comment by lllliufeng** [comment]
积回去（bushi

**Reply by Nie7bai** [reply]
难你天？

**Reply by zhixuan45** [reply]
好主意

**Comment by demolute** [comment]
下个火影就老实了

**Comment by real-strange** [comment]
少年，太虚了吧，平均一天两次就不行了<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可奸笑" width="75" >

**Comment by JIunA-F** [comment]
逆天

**Comment by locustguy** [comment]
牛啊

**Comment by AnonCN** [comment]
忍住，多忍忍，忍成习惯了就想不起来了
这是意志力的考验（
